.github/workflows/black.yml

```diff
@@ -0,0 +1,15 @@
+name: Check code format
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v3
```

Comment:
update this version, see the stac-data-tools PR discussions

Reply:
Fixed!